clang/lib/Driver/ToolChain.cpp

```diff
@@ -1290,6 +1296,9 @@ void ToolChain::AddCXXStdlibLibArgs(const ArgList &Args,
   case ToolChain::CST_Libstdcxx:
     CmdArgs.push_back("-lstdc++");
     break;
+
+  default:
```

Comment:
Let's spell out the CST_Stl case explicitly and avoid a default case. That way -Wswitch-enum will fire in the future if a new enum gets added.